### PR TITLE
ENCD-5356-add-examined-loci-on-FCC-experiments

### PR DESCRIPTION
**Pull request by zoldello**

### CODE REVIEW

File: src/encoded/static/components/experiment.js [hunk 1]
```diff
@@ -661,6 +662,25 @@ const ExperimentComponent = ({ context, auditIndicators, auditDetail }, reactCon
...
                             : null}
 
+                            {context.examined_loci && context.examined_loci.length > 0 ?
+                                <div data-test="examined-loci">
+                                    <dt>Examined Loci</dt>
+                                    <dd>
+                                        {/* A user can have a loci repeat. Therefore, uuid alone is not sufficient as an identifier */}
+                                        {context.examined_loci.map((loci, i) => (
+                                            loci.gene ?
+                                                <span key={[loci.gene.uuid, i].join('')}>
```
Comment (forresttanaka) [inline src/encoded/static/components/experiment.js]: A string template might look simpler, but probably makes no real difference.

File: src/encoded/static/libs/ordinal_suffix.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+ * @returns Number with ordinal suffix
+ */
+const getNumberWithOrdinal = (number) => {
+    const suffix = ['th', 'st', 'nd', 'rd'];
+    const percentage = number % 100;
+
+    return number + (suffix[(percentage - 20) % 10] || suffix[percentage] || suffix[0]);
+};
+
+export default getNumberWithOrdinal;
```
Comment (forresttanaka) [inline src/encoded/static/libs/ordinal_suffix.js:15]: Looks like no trailing carriage return?

**Review by forresttanaka — DISMISSED** [review]
The ordinal-suffix function is cool.

File: src/encoded/static/components/experiment.js (hunk 1 above)
Comment (zoldello) [reply src/encoded/static/components/experiment.js]: It was ES5 in me that creeped up. I updated this.

**Review by forresttanaka — APPROVED** [review]

**Pull request merged**